README.md

```diff
@@ -0,0 +1,64 @@
+# pySearch
+
+pySearch is a simple command line python script that allows for seamlessly performing a web search without removing your hands off the keyboard as it opens a new browser tab to search the input string. Normally when working on a command line interface, a user would need to use their mouse to perform a web search as they would take one hand off the keyboard and navigate to their browser. Even when using ALT+TAB, users would still need to navigate to the search bar with the mouse and then return to the keyboard to type their search. 
+
+pySearch simplifies this to a single command line command.
+
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Software you will need
+
+ Python - https://www.python.org/downloads/ 
+ Text Editor  - eg Visual Studio Code (if planning to edit this repo) 
+
+
+### Installing
+
+Clone this Repo  
+
+In Command Line Prompt 
+
+To run use 
+
+python C:\Users\pySearch\pySearch.py  ( different path name where you have cloned pySearch into) 
```

Comment:
Could you also list what all the command line arguments are? Note that the `-s` argument is required as it denotes your search. 

Run the script using `-h` to see a brief description of what each argument does. You can also read the code to see what the command line arguments do.

Take at the code or try running the code and calling the `-h`